### PR TITLE
Update README.md Example

### LINKED ISSUE #47: Clarifying real-life use-cases in Example
See https://github.com/paralleldrive/react-feature-toggles/pull/44

> Maybe ComingSoonComponent should be called HelpFallbackComponent or something similar, to make it more clear.

> In real use, I usually render entire pages as 404, or render nothing -- both of which users could configure as reusable fallback components that would work everywhere (like featureOr404(), featureOrNothing(), etc... I think the case where you render ComingSoon would be extremely rare. What you might render instead would be an older version of a component, like help404(HelpChatComponent, OldHelpComponent);

### PR DESCRIPTION
**Pull request by kennylavender**
resolves #47 
@thoragio, The prev example was pretty bad and confusing and since you have been busy with devanywehere I decided to just update it really quick.

Please review.

### CODE REVIEW

**Comment by ericelliott** [comment]
Can you push to the branch again to kick travis-ci awake?

File: README.md [hunk 1]
```diff
@@ -16,60 +16,52 @@ React Feature Toggles attempts to satisfy the following requirements:
...
+Setup withFeatures in the page HOC composition
 
-Let's create a help chat component placeholder to start; imagine it shows chat messages and what not.
 ```javascript
-// help-chat-component.js
-const HelpChat = () => <div>My real help chat stuff goes here...</div>;
-
-export default HelpChat;
+import { withFeatures } from '@paralleldrive/react-feature-toggles';
+import initalFeatures from '../config/initial-features';
```
Comment (thoragio) [inline README.md]: Typo: initalFeatures -> initialFeatures

**Comment by kennylavender** [comment]
Changes are not working in new branches either, will submit a travis help request.

**Comment by kennylavender** [comment]
Sent travis a message.

**Comment by thoragio** [comment]
@kennylavender You might want to try deleting this PR && using a new branch with the changes if you haven't tried that already. Might be quicker than waiting for a response from Travis, which generally takes a couple of days based on past experience troubleshooting similar issues.

**Comment by kennylavender** [comment]
@thoragio Travis answered, its because this repo switch to public now, so it has to be setup with travis-ci.org instead of travis-ci.com

**Comment by kennylavender** [comment]
Attempted to activate this repo on travis-ci.org but I do not have access.

@ericelliott can you allow this repo from travis-ci.org please.

screen shot 2017-10-17 at 5 55 53 am

**Comment by ericelliott** [comment]
Oops. Will fix.

**Comment by kennylavender** [comment]
@thoragio tests working now, ready for review.

**Pull request merged**